Library/test-helpers/lib.sh

```diff
@@ -1375,7 +1375,11 @@ limeCreateTestPolicy() {
     echo -e "${EXCLUDE}" >> excludelist.txt
 
     # create policy.json...
-    rlRun "limeCopyKeylimeFile --install cmd/convert_runtime_policy.py && python3 convert_runtime_policy.py -a allowlist.txt -e excludelist.txt -o policy.json"
+    rlRun "keylime_convert_runtime_policy -a allowlist.txt -e excludelist.txt -o policy.json"
```

Comment:
Could you please remove `rlRun` from this and following calls and possibly also connect those individual commands together and with the previous code through `&&`? We are calling this library function in tests from `rlRun` so better there are not two nested `rlRun`s used.